product_event_link/views/product_template_views.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" ?>
+<odoo>
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="name">product.template.common.form</field>
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product_auction.product_template_form_view" />
```

Comment:
We need include product_auction in the dependency, or find another hook point in product.product_template_form_view (should be the latter unless there is a reason to do otherwise, to avoid having more dependencies).  Please consult Kanda-san as necessary on this.